src/core/checkedint.d

```diff
@@ -682,29 +682,75 @@ unittest
 
 /// ditto
 pragma(inline, true)
+ulong mulu(uint x, ulong y, ref bool overflow)
+{
+    ulong r = x * y;
+    if (y >> 32 &&
+            r / y != x)
+        overflow = true;
+    return r;
+}
+
+/// ditto
+pragma(inline, true)
+ulong mulu(ulong x, uint y, ref bool overflow)
+{
+    return mulu(y, x, overflow);
+}
```

Comment:
I don't think `(ulong, uint, ref bool)` override is really needed, we don't do this for others.